### PR TITLE
Add missing flags explanation for the `kubectl gs template nodepool` command

### PR DESCRIPTION
**Pull request by marians**
Docs for `kubectl gs template nodepool` where missing some flags. This PR adds them.

### CODE REVIEW

File: src/content/reference/similar-ec2-instance-types/index.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ owner:
 
 # Similar EC2 Instance Types
 
+{{< platform_support_table aws="ga=11.2.0" >}}
+
```
Comment (marians) [inline src/content/reference/similar-ec2-instance-types/index.md:18]: This is unrelated to the main purpose of the PR. It will add this box:

image

**Comment by marians** [comment]
Pinging firecracker for review as the flags are AWS specific, with the exception of `--output`.

**Review by axbarsan — APPROVED** [review]

**Review by alex-dabija — APPROVED** [review]
